### DISCUSSION TITLE
FunkinSprite sucking

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#475] [Q&A] [answered]
bro I didn't even change anything 😭 
![image](https://github.com/user-attachments/assets/d77b5de8-d7c9-4e7f-a3a9-51ffe6adf79c)
![image](https://github.com/user-attachments/assets/127470a3-0362-40a4-a333-197512650013)

**Comment by r6915ee** [comment, marked as answer]
The devious, deliberate impostor `XmlAnimType` with the true and honest one `XMLAnimType`:

On a serious note, I guess it just has something to do with the capitalization, since it is likely case-sensitive. That's also not in the current commit so it may be an outdated version, so replace `XmlAnimType.NONE` with `XMLAnimType.NONE`.

**Reply by THEkatinamicrowave** [reply]
it did fix it
although I'm not sure how it got screwed up
last I checked, there aren't really any commits that edit FunkinSprite, nor did I edit it myself

we'll never know

**Reply by r6915ee** [reply]
This is an oddity to me as well. I don't really know how it decided to change, but it did.
